### ISSUE TITLE
version.json

### ISSUE THREAD

**Issue by newnon** [cocos2d/cocos2d-x-3rd-party-libs-bin#26]
why you don't update version.json "prebuilt_libs_version":"v3-deps-4", 
when download_deps.py check version it read this value and try to download deps every time(it useful for CI server)

**Comment by minggo** [comment]
Yep, it seems an error. But when i run `download-deps.py`, it will not download deps every time. 

Anyway, i will remember it in future.

**Comment by newnon** [comment]
Yes if i haven't removed old file it will not download it again but every time it unpack it.

**Comment by 1scaR1** [comment]
This issue is still be actual :(

**Comment by minggo** [comment]
Sorry about it, it is easy to forget to update the version.
